src/Compilers/Test/Core/TargetFrameworkUtil.cs

```diff
@@ -292,6 +293,7 @@ static TargetFrameworkUtil()
             TargetFramework.Net60 => ImmutableArray.CreateRange<MetadataReference>(LoadDynamicReferences("Net60")),
             TargetFramework.NetCoreApp or TargetFramework.Net70 => ImmutableArray.CreateRange<MetadataReference>(Net70.References.All),
             TargetFramework.Net80 => ImmutableArray.CreateRange<MetadataReference>(LoadDynamicReferences("Net80")),
+            TargetFramework.Net90 => ImmutableArray.CreateRange<MetadataReference>(LoadDynamicReferences("Net90")),
```

Comment:
Why are tests failling if this isn't used anywhere?

Example test failure in Microsoft.CodeAnalysis.CSharp.UnitTests.CollectionExpressionTests.SpanArgument_01(targetFramework: Net80):
 
```
++>   IL_001d:  newobj     "System.ReadOnlySpan<int?>..ctor(ref readonly int?)"
-->   IL_001d:  newobj     "System.ReadOnlySpan<int?>..ctor(in int?)"
```

Reply:
Hmm ... that is odd.

Reply:
The tests all pass when run individually but fail when the entire test class is executed. We must be holding onto state somewhere

Reply:
I do not see a single line in CollectionExpressionTests that starts with ```IL_001d:  newobj```. What Am I missing?

Regarding the diff, is it possible that we are updated to ref assembly for 8.0 that use ```ref readonly``` in the constructor?

Reply:
@jaredpar 

> The tests all pass when run individually but fail when the entire test class is executed. We must be holding onto state somewhere

It is likely that CI failures are expected given the changes to ref assemblies that you are making. It looks to me like your branch is behind main enough to make the difference between the local state and the merged state the CI is running against. That is why CI failures do not make sense in context of your local branch. Consider merging main into you branch, then running tests locally.

Reply:
Figured out what is happening here. When I updated the reference assemblies recently I missed updating the `net80` set. That means the NuPkg remained at 1.4.5 which is based off of the 8.0.0-preview.7.23375.6 version of .NET. The `ref readonly` parameter changes came in after that. Now that I unified the versions into a single MSBuild prop that dragged the `net80` version forward to RTM and we're seeing the signature differences.